torch/autograd/__init__.py

```diff
@@ -20,7 +20,7 @@
 from ..overrides import has_torch_function, handle_torch_function
 from . import functional
 from . import forward_ad
-from . import saved_variable_default_hooks as graph
```

Comment:
Somehow autodoc can't find the `graph` module if it's imported
with an alias.
https://www.internalfb.com/intern/opensource/ci/job/log/265181241677988/